examples/platformio/platformio.ini

```diff
@@ -0,0 +1,36 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+src_dir = .
+;lib_dir = ../..
+
+[debug]
+debug = ;-D DEBUG_NTPCLIENT
+
+[env]
+upload_speed = 921600
+monitor_speed = 115200
+lib_deps =
+    ESPNtpClient
```

Comment:
Wouldn't be better to reference here actual code instead Platformio Registry?

Reply:
Yes I agree, but you has it like that. I'm only copy the advanced example. But let me I do it.

Reply:
done